src/coreclr/vm/riscv64/asmhelpers.S

```diff
@@ -958,6 +958,22 @@ NESTED_ENTRY OnCallCountThresholdReachedStub, _TEXT, NoHandler
     EPILOG_BRANCH_REG t4
 NESTED_END OnCallCountThresholdReachedStub, _TEXT
 
+NESTED_ENTRY JIT_Patchpoint, _TEXT, NoHandler
+    PROLOG_WITH_TRANSITION_BLOCK
+
+    addi  a0, sp, __PWTB_TransitionBlock // TransitionBlock *
+    call  C_FUNC(JIT_PatchpointWorkerWorkerWithPolicy)
+
+    EPILOG_WITH_TRANSITION_BLOCK_RETURN
+NESTED_END JIT_Patchpoint, _TEXT
+
+// first arg register holds iloffset, which needs to be moved to the second register, and the first register filled with NULL
+LEAF_ENTRY JIT_PartialCompilationPatchpoint, _TEXT
+    addi    a1, a0, 0
+    addi    a0, zero, 0
+    b JIT_Patchpoint
```

Comment:
```suggestion
    mv    a1, a0
    li    a0, 0
    j JIT_Patchpoint
```

Reply:
I merged this manually into my changes, as I wanted to follow our standard of using C_FUNC for the tail jump.